src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -71,43 +75,20 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
-    private void execute_specifiedRemarkField_shouldModifyRemark(String remarkString) {
-        Remark remark = new Remark(remarkString);
-
-        Person personToBeEdited = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-
-        Person personToBeExpected = new Person(
-                personToBeEdited.getName(),
-                personToBeEdited.getPhone(),
-                personToBeEdited.getEmail(),
-                personToBeEdited.getAddress(),
-                personToBeEdited.getPriority(),
-                remark,
-                personToBeEdited.getTags());
-
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder()
-                .withRemark(remark.value)
-                .build();
-
-        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
-
-        Model expectedModel = new ModelManager(new AddressBook(model.getAddressBook()), new UserPrefs());
-        expectedModel.setPerson(personToBeEdited, personToBeExpected);
-
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS,
-                Messages.format(personToBeExpected));
-
-        assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
-    }
-
     @Test
     public void execute_specifiedRemarkField_shouldModifyRemark() {
-        execute_specifiedRemarkField_shouldModifyRemark("This is a remark!");
+        helper.execute_specifiedRemarkField_shouldModifyRemark("This is a remark!");
     }
 
     @Test
     public void execute_specifiedEmptyRemarkField_shouldModifyRemark() {
-        execute_specifiedRemarkField_shouldModifyRemark("");
+        helper.execute_specifiedRemarkField_shouldModifyRemark("");
+    }
+
+    @Test void execute_specifiedPriorityField_shouldModifyPriority() {
```

Comment:
Could you specify the access modifier and place the `@Test` annotation above the method signature?

Reply:
Noted. Thanks for catching that.